docs/book/src/developer/architecture/controllers/multi-tenancy.md

```diff
@@ -11,3 +11,108 @@ In order to support multi tenancy, the following rule applies:
 - Providers SHOULD deploy and run any kind of webhook (validation, admission, conversion)
   following Cluster API codebase best practices for the same release.
 - Providers MUST create and publish a `{type}-component.yaml` accordingly.
+- Providers MUST use the field of name `identityRef` to reference the multi-tenancy CR in the
+  `InfraCluster` CR.
+- Providers MUST implement `identityRef` must be of the following type:
```

Comment:
Do we (CAPO) have to implement the `identityRef` then, or is it enough that we reference on a single secret per namespace?
As per my understanding, this is only for the multi-tenancy secret reference, to enable the usage of multiple credentials.

Reply:
Updated